build.default.props

```diff
@@ -400,3 +411,33 @@ varnish.server.port = 8888
 # Drush Context configuration.
 # ----------------------------
 drush.db.dump = ${build.platform.dir}/dump.sql
+
+# Drush aliases configuration.
+# ----------------------------
+drush.alias.default = ${build.site}
+drush.alias.docker = docker
+
+# Drupal installation settings.
+# ----------------------------
+drupal.update.free.access = FALSE
+drupal.hash.salt =
+
+# Bactrack configuration
+# ----------------------------
+backtrac.file = ${project.tmp.dir}/bactrack.json
+
+
+# Toolkit location: ./includes/phing/props/drone.props
+# ------------------------------------------------------------------------------
+# Default properties for toolkit pipeline on Drone:
+# https://github.com/ec-europa/ne-pipelines/blob/master/drone/toolkit.yml
+# ------------------------------------------------------------------------------
+
+db.host = mysql
+solr.host = solr
+behat.base_url = http://web:8080
+behat.wd_host.url = http://selenium:4444/wd/hub
+share.path = /cache
+project.db.file = ./dump.sql
```

Comment:
Which database is this one?

Reply:
Used by default by Drone:
* https://github.com/ec-europa/ne-pipelines/blob/master/drone/test/toolkit.yml#L56
* https://github.com/ec-europa/toolkit/blob/master/includes/phing/props/drone.props